app/Providers/AuthServiceProvider.php

```diff
@@ -29,7 +30,14 @@ class AuthServiceProvider extends ServiceProvider
      *
      * @return void
      */
-    public function boot(): void {
+    public function boot(): void
+    {
         $this->registerPolicies();
+
+        Route::group(['prefix' => 'oauth', 'as' => 'oauth.'], function () {
+            Route::get('authorize', [AuthorizationController::class, 'authorize'])
+                ->middleware(['web'])
+                ->name('authorizations.authorize');
+        });
```

Comment:
Can you move this to the `routes/web.php`-file or is there a reason why the route is defined here?

Reply:
They need to be registered there, when they are defined in `routes/web.php` they do not correctly override the passport routes.

Reply:
Oh, okay! Good 2 know.